src/pretix/static/pretixbase/js/addressform.js

```diff
@@ -8,7 +8,8 @@ $(function () {
             url = this.getAttribute('data-country-information-url'),
             form = dependency.closest(".panel-body, form, .profile-scope"),
             isRequired = dependency.closest(".form-group").is(".required"),
-            dependents =  {
+            htmlRequired = dependency.closest(".no-required-attributes").length == 0,
```

Comment:
For my brain, it is easier to process it as an overwrite-option that forces anything to be optional. Might just be my brain …

```suggestion
            forceOptional = dependency.closest(".no-required-attributes").length > 0,
```

Should we add a comment, that plugins can add this CSS-class to make everything optional? Because in pretix itself, there is no use of this CSS-class elsewhere.